examples/rsvp-6-vevt/index.mjs

```diff
@@ -2,9 +2,10 @@ import { loadStdlib, test } from '@reach-sh/stdlib';
 import * as backend from './build/index.main.mjs';
 
 // Basics
+const GAS_LIMIT = 5000000;
 const stdlib = loadStdlib({ REACH_NO_WARN: 'Y' });
 const err = {
-  'ETH': 'transaction may fail',
+  'ETH': 'transaction failed',
   'ALGO': 'assert failed',
   'CFX': 'transaction is reverted',
```

Comment:
There's another error here:  https://output.circle-artifacts.com/output/job/97218047-c06c-4bcb-9a82-4d9ac415609f/artifacts/1/tmp/artifacts/CFX.rsvp-6-vevt.gz

Reply:
Receipt outcomeStatus is nonzero